packages/yoroi-extension/app/connector/components/signin/SignTxTabs.js

```diff
@@ -1,113 +1,128 @@
   const [value, setValue] = useState(0);
-  const match = useMediaQuery('(min-width: 700px)')
-  const handleChange = (event, newValue) => {
-    setValue(newValue);
-  };
+
+  const handleChange = (event, newValue) => setValue(newValue);
 
   const tabs = [
-    {
-      id: 0,
-      label: intl.formatMessage(globalMessages.overview),
-      component: overviewContent,
-    },
-    {
-      id: 1,
-      label: intl.formatMessage(messages.utxoAddresses),
-      component: utxoAddressContent,
-    },
+    { id: 0, label: intl.formatMessage(messages.details), component: detailsContent },
+    { id: 1, label: intl.formatMessage(messages.utxos), component: utxosContent },
+    { id: 2, label: intl.formatMessage(messages.connection), component: connectionContent },
   ];
 
+  const isTestEnv = environment.isNightly() || environment.isTest();
+
   return (
     <Background>
+      <Typography color="#242838" variant="h4" align="center" my="32px">
+        {intl.formatMessage(connectorMessages.signTransaction)}
+      </Typography>
       <TabContext value={value}>
         <Box
-          sx={match && {
+          sx={{
             backgroundColor: 'var(--yoroi-palette-common-white)',
-            boxShadow: '0 4px 6px 0 #DEE2EA, 0 1px 2px 0 rgba(222,226,234,0.82), 0 2px 4px 0 rgba(222,226,234,0.74)',
-            heigh: '100%',
+            marginLeft: '32px',
+            marginRight: '32px',
           }}
         >
           <TabList
             sx={{
-              '& .MuiTabs-indicator': { height: '4px' },
-              width: match ? '600px' : 'auto',
-              margin: 'auto',
+              '& .MuiTabs-indicator': { height: '2px' },
+              width: '480px',
               boxShadow: 'none',
             }}
+            textColor="primary"
+            indicatorColor="primary"
             onChange={handleChange}
             aria-label="Staking tabs"
           >
-            {tabs.map(({ label, id }) => (
-              <StyledTab label={label} value={id} />
-            ))}
+            {tabs.map(
+              ({ label, component, id }) =>
+                component !== null && (
+                  <StyledTab
+                    key={id}
+                    disableRipple
+                    label={
+                      <Typography variant="body1" fontWeight={500}>
+                        {label}
+                      </Typography>
+                    }
+                    value={id}
+                  />
+                )
+            )}
           </TabList>
+          <div style={{ backgroundColor: '#DCE0E9', height: '1px', width: '100%' }} />
         </Box>
-        {tabs.map(({ component, id }) => (
-          <TabPanel
-            sx={{
-              height: environment.isNightly() || environment.isTest() ? 'calc(84vh - 55px)': 'calc(100vh - 103px)', // 103 = 52px (topbar) + 51px (tabs bar)
-              overflowY: 'scroll',
-              width: match ? '600px' : 'auto',
```

Comment:
Why are we removing the media query?

Reply:
Design spec. Check it out on figma! :)

Reply:
@nistadev  the design spec doesn't mention the view on large screens